packages/ui/src/apis/interaction.ts

```diff
@@ -200,3 +200,21 @@ export const bindSocialRelatedUser = async (payload: SocialEmailPayload | Social
 
   return api.post(`${interactionPrefix}/submit`).json<Response>();
 };
+
+export const linkWithSocial = async (connectorId: string) => {
+  // Replace email/phone profile with connectorId. User previous verified email/phone sign-in instead
```

Comment:
nit: I don't quite get this comment, especially the second sentence.

Reply:
The previous condition is registered using the connector as an identifier and trying to fulfill a new email/phone. Here we need to switch the logic, and sign in with an existing verified email/phone and link the connector instead.

Reply:
I'll update the comment to be more clear.

Reply:
So is there a typo? `User previous verified email/phone...` should be `Use previously verified email/phone...`

Reply:
Ah, didn't notice that. should be.